CONTRIBUTING.md

```diff
@@ -17,6 +17,27 @@ $ docker-compose up
 
 Go to https://127.0.0.1:8888/ to access temboard runing with your code!
 
+## CSS
+
+Temboard UI mainly relies on `Bootstrap`. The CSS files are compiled with
+`SASS`.
+
+This task requires you to have [Ruby](http://www.ruby-lang.org/en/downloads/)
```

Comment:
Do you know https://pypi.python.org/pypi/libsass ? It's pretty fast and should be compatible ! It uses libsass C++ implementation from sass authors !

``` console
$ pip install libsass
$ sassc --help
Usage: sassc [options] SCSS_FILE [OUT_CSS_FILE]

Options:
  --version             show program's version number and exit
  -h, --help            show this help message and exit
  -t STYLE, -s STYLE, --style=STYLE, --output-style=STYLE
                        Coding style of the compiled result.  Choose one of
                        compact, expanded, compressed, or nested. [default:
                        nested]
  -m, -g, --sourcemap   Emit source map.  Requires the second argument (output
                        css filename).
  -I DIR, --include-path=DIR
                        Path to find "@import"ed (S)CSS source files.  Can be
                        multiply used.
  -w, --watch           Watch file for changes.  Requires the second argument
                        (output css filename).
  -p PRECISION, --precision=PRECISION
                        Set the precision for numbers. [default: 5]
  --source-comments     Include debug info in output
```